bins/ggipcd/src/services/config/get_configuration.c

```diff
@@ -71,16 +88,22 @@ GglError ggl_handle_get_configuration(
         component_name, key_path_obj->list, &full_key_path
     );
     if (ret != GGL_ERR_OK) {
+        *ipc_error = (GglIpcError
+        ) { .error_code = GGL_IPC_ERR_SERVICE_ERROR,
+            .message = GGL_STR("Failed to make config path object.") };
         return ret;
     }
 
     GglObject read_value;
     ret = ggl_gg_config_read(full_key_path, alloc, &read_value);
     if (ret != GGL_ERR_OK) {
+        if (ret == GGL_ERR_NOENTRY) {
+            *ipc_error
+                = (GglIpcError) { .error_code = GGL_IPC_ERR_RESOURCE_NOT_FOUND,
+                                  .message = GGL_STR("Key not found") };
```

Comment:
Seems some IPC error messages have periods and some don't? Lets match Classic on that.